src/stores/SettingsController.ts

```diff
@@ -1,43 +0,0 @@
-import ini from 'ini';
-
-import { Settings, useSettingsStore } from '#src/stores/SettingsStore';
-import { logDev } from '#src/utils/common';
-import { OTT_GLOBAL_PLAYER_ID } from '#src/config';
-
-export const initSettings = async () => {
```

Comment:
Maybe this whole thing can just be a useSettings hook?

Reply:
We will use a pluggable "Settings" service.